README.md

```diff
@@ -0,0 +1,116 @@
+<div id="top"></div>
+
+**The project is no longer maintained. Sorry.**
+
+<!-- TABLE OF CONTENTS -->
+<details>
+  <summary>Table of Contents</summary>
+  <ol>
+    <li>
+      <a href="#about-the-project">About The Project</a>
+      <ul>
+        <li><a href="#built-with">Built With</a></li>
+      </ul>
+    </li>
+    <li>
+      <a href="#getting-started">Getting Started</a>
+      <ul>
+        <li><a href="#prerequisites">Prerequisites</a></li>
+        <li><a href="#installation">Installation</a></li>
+      </ul>
+    </li>
+    <li><a href="#usage">Usage</a></li>
+    <li><a href="#acknowledgments">Acknowledgments</a></li>
+  </ol>
+</details>
+
+<!-- ABOUT THE PROJECT -->
+
+## About The Project
+
+The goal of this project is to automate the manual labor of valuable data collection for over 100 workers in the district education system.
+
+The web application provide set of tools to create/manipulate/distribute questionnaires of any kind as well as collect/view/export answers.
+
+<p align="right">(<a href="#top">back to top</a>)</p>
+
+### Built With
+
+- [React.js](https://reactjs.org/)
+- [Bootstrap](https://getbootstrap.com)
+- [Express](https://expressjs.com/)
+- [PostgreSQL](https://www.postgresql.org/)
+- [Webpack](https://webpack.js.org/)
+
+<p align="right">(<a href="#top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+This is an example of how to list things you need to use the software and how to install them.
+
+- node: `~8.17.0` (`lts/carbon`)
+- npm: `~6.13.4`
+- postgresql: `~10.14`
+
+You may want to use [nvm](https://github.com/nvm-sh/nvm) package to use old node version.
+
+#### \* Apple silicon chips troubleshooting
+
+If you are using MacBook with silicon chip you may have a hard time installing `node v8`.
```

Comment:
I guess, all chips are silicon, technically. Maybe it's worth specifying that it's M1?

Reply:
Fixed.